tests/modbus/suricata.yaml

```diff
@@ -0,0 +1,9 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: true
+      types:
+        - modbus
```

Comment:
Why do not we need to set `app-layer.protocols.modbus.enabled` ?

Reply:
I think it's because it's enabled by default in [modbus.rs](https://github.com/OISF/suricata/pull/6090/files#diff-57a6cf267e4fb39aa835b56cc817f0c7dccbafc37de20f4c90eaf8eb216a5958R543):

```rust
// enable_default: 1
AppLayerRegisterProtocolDetection(&parser, 1);
```